examples/example-1/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "example-1",
+  "version": "1.0.0",
+  "description": "example broccoli-eyeglass project 1",
+  "main": "Brocfile.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Alan Hogan",
+  "license": "SAX-PD",
```

Comment:
can you either remove the license or change this to `Apache-2.0` to align with the global license of the project?